src/app/tests/integration/chip_im_responder.cpp

```diff
@@ -98,11 +100,44 @@ void DispatchSingleClusterCommand(chip::ClusterId aClusterId, chip::CommandId aC
 } // namespace chip
 
 namespace {
-
-// The CommandHandler object
 chip::TransportMgr<chip::Transport::UDP> gTransportManager;
 chip::SecureSessionMgr gSessionManager;
-chip::SecurePairingUsingTestSecret gTestPairing;
+chip::SecurePairingUsingTestSecret gTestPairing(chip::Optional<chip::NodeId>::Value(chip::kUndefinedNodeId),
+                                                static_cast<uint16_t>(0), static_cast<uint16_t>(0));
+LivenessEventGenerator gLivenessGenerator;
+
+uint64_t gDebugEventBuffer[256];
```

Comment:
why uint64_t instead of uint8_t and increase in buffer size? Are there alignment requirements? if so it may be clearer if they are called out as 'alignas'

Reply:
no alignment requirement, just use uint8_t, thanks